src/pages/images/actions/ExportImageTarballBtn.tsx

```diff
@@ -0,0 +1,54 @@
+import { FC, useState } from "react";
+import { LxdImage } from "types/image";
+import { ActionButton, Icon } from "@canonical/react-components";
+import { useToastNotification } from "context/toastNotificationProvider";
+
+interface Props {
+  image: LxdImage;
+}
+
+const ExportImageTarballBtn: FC<Props> = ({ image }) => {
+  const toastNotify = useToastNotification();
+  const [isLoading, setLoading] = useState(false);
+  const description = image.properties?.description ?? image.fingerprint;
+  const isUnifiedTarball = image.update_source == null; //Only Split Tarballs have an update_source.
+
+  const handleExport = (image: LxdImage) => {
```

Comment:
```suggestion
  const handleExport = () => {
```
We don't need to pass the argument, it is available through the props.